src/language-configuration.js

```diff
@@ -8,6 +8,15 @@ module.exports = {
     ["[", "]"],
     ["<", ">"]
   ],
+  "colorizedBracketPairs": [
+    ["{", "}"],
+    ["{{", "}}"],
+    ["{{-", "-}}"],
+    ["{%", "%}"],
+    ["{%-", "-%}"],
```

Comment:
FYI it's also possible to use `["{%-", "%}"]`, `["{%", "-%}"]`, `["{{", "-}}"]`, and `["{{", "-}}"]`. I use these combinations actually.

For example, user might want to remove all the preceding whitespace, but not following whitespace:
This is the code that I wrote yesterday:

```liquid
{% comment -%}
    This sets meta description for every Policy page. Policy pages don't have meta descriptions by default.
  {%- endcomment -%}
  {%- if request.page_type == 'policy' -%}
    {%- for policy in shop.policies -%}
      {%- if policy.title == page_title -%}
        {%- assign policy_page = policy -%}
      {%- endif -%}
    {%- endfor -%}
    <meta name="description" value="{{ policy_page.body | strip_html | truncate: 150, '' }}">
  {%- endif %}
```